### PR TITLE
feat:  add device validate

### LINKED ISSUE #125: Add device validate

### PR DESCRIPTION
**Pull request by FelixMZ2018**
feat:  add device validate

### CODE REVIEW

File: src/dataPoints/downloadRequest/createForProcess.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const { DEFAULT_REQ_OPTS } = require('../../utils/defaultReqOpts')
  * @memberof nexus.dataPoints.downloadRequest
  * @function createForProcess
  * @async
- * @param {string} processId
+ * @param {string} processId the id of the process
```
Comment (FelixMZ2018) [inline src/dataPoints/downloadRequest/createForProcess.js]:
```suggestion
 * @param {string} processId
```

File: src/device/validate.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const { default: API } = require('@aws-amplify/api-rest')
+
+const serializeAxiosResponse = require('../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../utils/defaultReqOpts')
+
+/**
+ * @desc returns limited information to validate a device after installation
```
Comment (FelixMZ2018) [inline src/device/validate.js]:
```suggestion
 * @desc returns limited device data 
 ```

File: test/src/device/validate.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const faker = require('faker')
+const { default: API } = require('@aws-amplify/api-rest')
+
+const validate = require('device/validate')
+const { DEFAULT_REQ_OPTS } = require('utils/defaultReqOpts')
+const testDefaultHTTPResponses = require('../../testHelpers/testDefaultHTTPResponses')
+
+describe('get', () => {
```
Comment (FelixMZ2018) [inline test/src/device/validate.test.js]:
```suggestion
describe('validate', () => {
```

**Review by cairesana — APPROVED** [review]

**Comment by lev-leontyev-ospin** [comment]
what's the status of this issue?

**Pull request merged**